src/layout/layout.module.css

```diff
@@ -0,0 +1,40 @@
+.wrapper {
+    height: 100%;
+    display: grid;
+    grid-template-rows: auto 1fr;
+    grid-template-columns: 1fr 0;
+    grid-template-areas:
+        'header header'
+        'main   sidebar';
+    transition: grid-template-columns 200ms ease-out;
```

Comment:
Currently this works in FF only, other browser have not implemented the spec yet (see this stackoverflow reply).

@cooper-joe I've talked to @ismay about this and our perspective is to ignore it as it'll be implemented at some point in the future. We don't know when though. If this is crucial functionality, then @ismay could change it and implement a more complicated solution to get this to work.

The current implementation is like this:
```
.layout (display: grid)
  > .header
  > .workspace
  > .rightHandSidebar
  > .bottomBar
```

The alternative solution would look similar to this:
```
.layout (display: flex | grid)
  > header
  > main (display: flex)
    > .workspace (display: flex; flex-direction: columns; flex-grow: 1)
      > .data (flex-grow: 1)
      > .bottomBar (fixed height)
    > .rightHandPanned (fixed width with transition from 0 to fixed and vice versa)
```

Reply:
One note is that it _might_ be possible to get the animation with a more complex solution. In trying to get there whilst working on this PR though, I found that it led to other regressions.

Reply:
So if it's absolutely crucial we could give it another shot, see if we've overlooked a solution. But if it's not crucial then the existing solution would be a simple and robust way to achieve the specced layout (with the only exception being the lack of animations).

Reply:
Animation is a supporting element here, so I don't think it warrants sacrificing the simple, robust layout. The design, user flow, and concept are all still valid without the animation, we just lose a little bit of _smoothness_. But, on balance, I'm OK with this solution.

Reply:
All right, I'll remove the animation and keep the layout as is then. By the way @cooper-joe, there are still other options if we want to make the transition less jarring. We could fade the elements in and out for example. Or use GPU scaling for the size transition. We could talk about that after this PR, I think we could make something work.